src/index.tsx

```diff
@@ -1,10 +1,40 @@
-import React from "react";
+import * as React from "react";
 import ReactDOM from "react-dom/client";
+import styled from "styled-components";
+import { createGlobalStyle } from 'styled-components';
+import { reset } from 'styled-reset';
 
+
+const MainStyle = createGlobalStyle`
+  ${reset}
+
+  html {
+    box-sizing: border-box;
+    font-family: 'Covered By Your Grace', cursive;
+  }
+  *, *:before, *:after {
+    box-sizing: inherit;
+  }
+`;
+
+const styles = {
+  container: {
+    height: '100vh',
+    width: '100vw',
+    margin: 0,
+    padding: '20px',
+    backgroundColor: '#334445',
+    color: '#F5D547',
+  },
+};
 const HomePage = () => {
-  return <div>
-    Home
+
+return( 
+  <div style={styles.container}>
```

Comment:
please stick to styled-components, mixing two types of styling makes it hard to reason about how stlying is handled

Reply:
styles.container is referencing the const that is described which sets the background fill, as well as setting text padding, and colour. Haven't yet changed this part, but I will double-check styled styled-components for if there is a built-in edit for the background colour and making sure the back fill goes edge to edge.